testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -311,7 +313,16 @@ impl SignerTest<SpawnedSigner> {
         //  whenever it has crossed the threshold.
         assert!(signature.len() >= num_signers * 7 / 10);
 
-        let reward_cycle = self.get_current_reward_cycle();
+        let new_reward_cycle = self.get_current_reward_cycle();
+        let reward_cycle = if new_reward_cycle != old_reward_cycle {
+            old_reward_cycle
+        } else {
+            new_reward_cycle
+        };
```

Comment:
This doesn't seem right. Isn't this the same as 
```rust
let reward_cycle = old_reward_cycle;
```

Reply:
Yes this is intentional. At the boundary, use the old reward cycle.

Reply:
Done.